site/app/controllers/admin/PlagiarismController.php

```diff
@@ -489,6 +474,33 @@ public function reRunPlagiarism($gradeable_id) {
         $this->core->redirect($this->core->buildCourseUrl(['plagiarism']) . '?' . http_build_query(['refresh_page' => 'REFRESH_ME']));
     }
 
+
+    /**
+     * @Route("/courses/{_semester}/{_course}/plagiarism/configuration/new", methods={"GET"})
+     */
+    public function configureNewGradeableForPlagiarismForm() {
+        $semester = $this->core->getConfig()->getSemester();
+        $course = $this->core->getConfig()->getCourse();
+        $gradeable_with_submission = array_diff(scandir("/var/local/submitty/courses/$semester/$course/submissions/"), ['.', '..']);
```

Comment:
Could replace all the usages of `/var/local/submitty/courses/$semester/$course` with `$this->core->getConfig()->getCoursePath()`